app/src/main/java/com/intervaltimer/google/activities/WorkoutActivity.java

```diff
@@ -162,17 +170,49 @@ public void onClick(View view) {
         //getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     }
 
+    private void resumeTimer() {
+        invertFabSelection();
+        if (timerService!= null) {
```

Comment:
"timerService!=" vs "timerService !=" with a space?

Let's follow up on the linter in our stand up...